lib/internal/event_target.js

```diff
@@ -730,14 +739,21 @@ class EventTarget {
     let handler = root.next;
     let next;
 
-    while (handler !== undefined &&
-           (handler.passive || event?.[kStop] !== true)) {
+    const iterationCondition = () => {
+      if (root.resistStopPropagation) {
```

Comment:
nit:
```suggestion
      if (handler === undefined) {
        return false;
      }
      return root.resistStopPropagation || handler.passive || event?.[kStop] !== true;
```
sorry for the non-multi-line suggestion, but it might be cleaner to check handler first, and then only branch later?

Reply:
Looks way better, thanks 🙂